start

```diff
@@ -264,7 +290,7 @@ function validateHorizon () {
 
 function validateSorobanRPC () {
     COUNTER=1
-    while ! (curl --silent --location --request POST 'http://localhost:8000/soroban/rpc' \
+    while ! (curl --silent --location --request POST "$TARGET_NETWORK_RPC_URL" \
```

Comment:
would it be possible to redirect the stderr from the curl into /dev/null ?
The reason I'm asking is that before the point where curl is providing "success" case, it would produce several fail cases. in these cases, the jq is going to return an error, since it wasn't able to process the output correctly.
few options:
maybe run curl -o <filename> && cat <filename> | jq ... ?

Reply:
yes, good point, I tried redirect on `jq` only and it worked - https://github.com/stellar/system-test/pull/13/commits/f02be6c893b3a63bf41c7f9e1451ef53dc62e416